### PR TITLE
update HAS owners file and rbac

### PR DESCRIPTION
**Pull request by yangcao77**
update HAS owners file and rbac

### CODE REVIEW

**Review by thepetk — APPROVED** [review]

**Comment by yangcao77** [comment]
@thepetk May I get another lgtm for this PR?

**Review by thepetk — APPROVED** [review]
/lgtm after all tests are passed

**Pull request merged**
